uui-editor/src/plugins/uploadFilePlugin/file_uploader.ts

```diff
@@ -24,22 +27,25 @@ type UploadFile = (
 
 const UPLOAD_BLOCKS = {
     attachment: (file: FileUploadResponse) => ({
-        type: ATTACHMENT_PLUGIN_TYPE,
+        type: ATTACHMENT_TYPE,
         data: {
-            ...file,
+            ...file, // TODO: remove, seems irrelevant
             fileName: file.name,
         },
         children: [{ text: '' }],
     }),
-    image: (file: FileUploadResponse) => ({
-        type: IMAGE_PLUGIN_TYPE,
-        data: file,
-        url: file.path,
+    image: (file: FileUploadResponse): TImageElement => ({
+        type: IMAGE_TYPE,
+        data: {
+            ...file, // TODO: remove, seems irrelevant
+            size: {},
```

Comment:
why do we replace size filed to empty object? Also, comments about removing file from data looks outdated regarding our previous sync up.

Reply:
The fields seems to completely irrelevant except `path` for iframe / image and `name` of the file for attachment. Other fields should be skipped to keep content clean.

Reply:
Discussed offline